doc/MISSIONS_JSON.md

```diff
@@ -126,6 +126,16 @@ Alternately, if present, it can be an object as described below.
 ### start / end / fail effects
 If any of these optional fields are present they can be objects with the following fields contained:
 
+### origin
+This determines how the player can be given this mission. There are a number of different options for this as follows.
+
+string ID             | Usage
+`ORIGIN_GAME_START`   | Given when the game starts
+`ORIGIN_OPENER_NPC`   | NPC comes up to you when the game starts
+`ORIGIN_ANY_NPC`      | Any NPC
+`ORIGIN_SECONDARY`    | Given at the end of another mission
+`ORIGIN_COMPUTER`     | Taken after reading investigation provoking entries in computer terminal
```

Comment:
Might need a bit of extra markdown so that Github displays it as a table:

```suggestion
| string ID             | Usage
| ---                   | ---
| `ORIGIN_GAME_START`   | Given when the game starts
| `ORIGIN_OPENER_NPC`   | NPC comes up to you when the game starts
| `ORIGIN_ANY_NPC`      | Any NPC
| `ORIGIN_SECONDARY`    | Given at the end of another mission
| `ORIGIN_COMPUTER`     | Taken after reading investigation provoking entries in computer terminal
```

Reply:
Thanks!